### PR TITLE
LSR-1103 Adapt Cookie login to the last version of the mobile-sdk (Android)

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by dgarciasarai** [comment]
@nhpatt please review and merge after that :)

File: android/library/src/main/java/com/liferay/mobile/screens/auth/login/LoginScreenlet.java [hunk 1]
```diff
@@ -218,7 +241,8 @@ protected View createScreenletView(Context context, AttributeSet attributes) {
 
 		loginViewModel.setAuthenticationType(authenticationType);
 
-		if (authenticationType.equals(AuthenticationType.BASIC) || authenticationType.equals(AuthenticationType.COOKIE)) {
+		if (authenticationType.equals(AuthenticationType.BASIC) || authenticationType.equals(
```
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/auth/login/LoginScreenlet.java]: Invert the if

File: android/library/src/main/java/com/liferay/mobile/screens/auth/login/interactor/LoginCookieInteractor.java [hunk 2]
```diff
@@ -22,9 +23,17 @@ public BasicEvent execute(Object... args) throws Exception {
 		String password = (String) args[1];
 		Authenticator authenticator = (Authenticator) args[2];
 
+		boolean shouldHandleExpiration = (boolean) args[3];
```
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/auth/login/interactor/LoginCookieInteractor.java]: Does it makes sense to make them required in every call?

File: android/library/src/main/java/com/liferay/mobile/screens/auth/login/LoginScreenlet.java (hunk 1 above)
Comment (victorg1991) [reply android/library/src/main/java/com/liferay/mobile/screens/auth/login/LoginScreenlet.java]: Done :D

File: android/library/src/main/java/com/liferay/mobile/screens/auth/login/interactor/LoginCookieInteractor.java (hunk 2 above)
Comment (victorg1991) [reply android/library/src/main/java/com/liferay/mobile/screens/auth/login/interactor/LoginCookieInteractor.java]: This is the only way to configure this attribute, what do you suggest?

**Pull request merged**